integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseTableOperations.java

```diff
@@ -288,9 +295,10 @@ static Mono<GetTableResponseBody> handleCreateUpdateHttpError(
       return Mono.error(new WebClientResponseWithMessageException((WebClientResponseException) e));
```

Comment:
shall we get rid of this block?

Reply:
This one can still be a non CSUE? Gor example forbidden or unauthorized could still be returned as such

Reply:
they will be categories as commit-failure in the caller of `doCommit` and we should be fine ?

Reply:
yes. Because these errors should happen only before HTS commit